apm-sniffer/apm-toolkit-activation/apm-toolkit-trace-context-activation/src/main/resources/skywalking-plugin.def

```diff
@@ -1 +1 @@
-org.skywalking.apm.toolkit.activation.trace.TraceContextActivation
\ No newline at end of file
+tracecontext-3.x=org.skywalking.apm.toolkit.activation.trace.TraceContextActivation
```

Comment:
Where does `3.x` comes from? Our project version?  Same as opentracing-toolkit-activation, just `tracecontext` is enough.